%!TEX TS-program = xelatex
\documentclass{update_nihbiosketch}
\usepackage{xspace}
%\usepackage{draftwatermark}  % delete this in your document!
%\SetWatermarkText{Sample}    % delete this in your document!
%\SetWatermarkLightness{0.9}  % delete this in your document!

%------------------------------------------------------------------------------

% Biological Abbreviations
\newcommand\dros{{\itshape Drosophila}\xspace}
\newcommand\dmel{{\itshape D.~melanogaster}\xspace}
\newcommand\scer{{\itshape S.~cerevisiae}\xspace}
\newcommand\saccer{{\itshape Saccharomyces cerevisiae}\xspace}
\newcommand\xenopus{{\itshape Xenopus}\xspace}
\newcommand\invitro{{\itshape in~vitro}\xspace}
\newcommand\invivo{{\itshape in~vivo}\xspace}
\newcommand\ten[1]{$\times$10$^{#1}$}
\newcommand\panc{Panc\dash1\xspace}
\newcommand\orcmt{\emph{orc1\dash 161}\xspace}

% Other Abbreviations
\newcommand\eg{\emph{e.g.}\xspace}
\newcommand\dash{\nobreakdash-\hspace{0pt}}

\name{MacAlpine, David M.}
\eracommons{MACALP001}
\position{Professor of Pharmacology and Cancer Biology}

\begin{document}
%------------------------------------------------------------------------------

\begin{education}
Texas A\&M University, College Station, TX  & BS          & 05/1993   & Biochemistry \\
Texas A\&M University, College Station, TX & MS        & 05/1995  & Genetics \\
University of Texas Southwestern Med. Ctr., Dallas, TX  & PhD  &05/2001 & Genetics\\
Massachusetts Institute of Technology, Cambridge, MA  & Postdoctoral & 09/2006 & Biochemistry \\
\end{education}


\section{Personal Statement}
\begin{statement}
\addtolength{\parskip}{1.5mm}
\noindent As a Professor of Pharmacology and Cancer Biology, my research group at Duke University adopts a multidisciplinary approach, intersecting genomic studies with advanced computational methods. We've developed innovative genomic techniques to unravel chromatin dynamics, focusing on how local chromatin environments impact DNA replication\textsuperscript{1}, repair\textsuperscript{2}, and transcription\textsuperscript{3}. This work is critical for understanding the inheritance of epigenetic information and the underlying mechanisms ensuring genetic stability.

\noindent My laboratory's strength lies in blending experimental science with computational analysis. I am adept in cloud computing, utilizing platforms such as Azure, AWS, and GCP, which has been instrumental in our research. 
 I was also a co-PI for NHGRI's modENCODE/ENCODE Data Analysis Center and have experience analyzing and integrating diverse genomic datasets to derive high impact discoveries\textsuperscript{4}.  
 
\noindent I take pride in my commitment to open science. Our public GitHub repository ensures full transparency and reproducibility of our research. This repository includes data and code for the majority of our publications, allowing peers  to replicate and build upon our work.

\noindent In my educational endeavors, I focus on bridging gaps between experimental biology and computational analysis. I teach a specialize course in computational biology for experimental biologists, emphasizing rigorous and reproducible research with large genomic datasets.  I have also pioneered the use of Jupyter notebooks in the classroom, integrating them with virtual Docker containers managed from Duke's private GitHub/GitLab repository. This innovative approach, developed in collaboration with Duke's Office of Information Technology, enhances hands-on learning and has been well-received by students with little to no prior training in computational biology.

\noindent As Director of Graduate Studies in Pharmacology, I am deeply involved in nurturing a diverse and vibrant community of graduate students. My role encompasses recruitment, mentorship, and academic guidance, ensuring a supportive and enriching environment for their professional growth. Additionally, my work extends to promoting diversity university-wide, as a faculty mentor for Duke's BioCORE program and a member of the selection committee for the Dean's Graduate Fellowships.

\noindent Beyond my institutional roles, I contribute actively to the broader research community. I am on the editorial board of \textit{Genome Research} and regularly review grants for the American Cancer Society, National Science Foundation, and the National Institutes of Health.

\noindent My overarching goal is to continue fostering an environment of collaborative learning, innovative research, and dedication to the advancement of science and education.


% \noindent My research group employs a multidisciplinary strategy to explore how local chromatin environments influence core processes like DNA replication\textsuperscript{1}, repair\textsuperscript{2}, and transcription\textsuperscript{3}. We have pioneered novel genomic approaches to study chromatin dynamics and profile chromatin assembly post-replication to elucidate the inheritance of epigenetic information\textsuperscript{4}. Our work melds experimental and computational sciences to decode chromatin-mediated mechanisms that ensure genetic and epigenetic inheritance. My paramount training responsibility is mentoring a small research group comprised of both experimental and computer scientists. Our hands-on training enables my trainees to generate and rigorously interpret extensive genomic data sets, which has fostered their successful careers in both academia and industry.

% \noindent As Director of Graduate Studies in Pharmacology, I play a pivotal role in promoting the growth of a diverse cohort of graduate students. I am responsible for overseeing recruitment, providing academic and professional advice, and ensuring our students feel supported and motivated throughout their graduate journey. My dedication to fostering diversity extends beyond my laboratory and the pharmacology program; I am a faculty mentor for Duke's  BioCORE program and I also serve on the selection committee for the Dean's Graduate Fellowships which are aimed at promoting diversity across the entire university.  

% \noindent My commitment to education extends into the classroom where I teach various courses, including a specialized course on computational biology for experimental biologists. This course provides essential training in rigor and reproducible research involving large genomic datasets, preparing students to conduct responsible, reproducible, and high-quality research.

% \noindent Finally, I am an active participant in the research community. I serve on the editorial board of \textit{Genome Research} and frequently review grants for the American Cancer Society, National Science Foundation, and the National Institutes of Health.  

\begin{enumerate}

\item Hoffman RA, MacAlpine HK, \textbf{MacAlpine DM}. Disruption of origin chromatin structure by helicase activation in the absence of DNA replication. Genes Dev. 2021 Oct 1;35(19-20):1339-1355. doi: 10.1101/gad.34857.121. Epub 2021 Sep 23. PMC8494203.

\item Tripuraneni V, Memisoglu G, MacAlpine HK, Tran TQ, Zhu W, Hartemink AJ, Haber JE, \textbf{MacAlpine DM}. Local nucleosome dynamics and eviction following a double-strand break are reversible by NHEJ-mediated repair in the absence of DNA replication. Genome Res. 2021 May;31(5):775-788. doi: 10.1101/gr.271155.120. Epub 2021 Apr 2. PMC8092003.

\item Tran TQ, MacAlpine HK, Tripuraneni V, Mitra S, \textbf{MacAlpine DM}*, Hartemink AJ. Linking the dynamics of chromatin occupancy and transcription with predictive models. Genome Res. 2021 Jun;31(6):1035-1046. doi: 10.1101/gr.267237.120. Epub 2021 Apr 23. PMC8168580. *Co-corresponding author

\item Ho JW, Jung YL, Liu T, Alver BH, Lee S, Ikegami K, Sohn KA, Minoda A, Tolstorukov MY, Appert A, Parker SC, Gu T, Kundaje A, Riddle NC, Bishop E, Egelhofer TA, Hu SS, Alekseyenko AA, Rechtsteiner A, Asker D, Belsky JA, Bowman SK, Chen QB, Chen RA, Day DS, Dong Y, Dose AC, Duan X, Epstein CB, Ercan S, Feingold EA, Ferrari F, Garrigues JM, Gehlenborg N, Good PJ, Haseley P, He D, Herrmann M, Hoffman MM, Jeffers TE, Kharchenko PV, Kolasinska-Zwierz P, Kotwaliwale CV, Kumar N, Langley SA, Larschan EN, Latorre I, Libbrecht MW, Lin X, Park R, Pazin MJ, Pham HN, Plachetka A, Qin B, Schwartz YB, Shoresh N, Stempor P, Vielle A, Wang C, Whittle CM, Xue H, Kingston RE, Kim JH, Bernstein BE, Dernburg AF, Pirrotta V, Kuroda MI, Noble WS, Tullius TD, Kellis M, \textbf{MacAlpine DM}*, Strome S, Elgin SC, Liu XS, Lieb JD, Ahringer J, Karpen GH, Park PJ. Comparative analysis of metazoan chromatin organization. Nature. 2014 Aug 28;512(7515):449-52. PMC4227084. * Co-corresponding author
%\item Guti√©rrez MP, MacAlpine HK, \textbf{MacAlpine DM}. Nascent chromatin occupancy profiling reveals locus- and factor-specific chromatin maturation dynamics behind the DNA replication fork. Genome Res. 2019 Jul;29(7): 1123-1133. doi: 10.1101/gr.243386.118. Epub 2019 Jun 19. PMC6633257.

%\item Chen B, MacAlpine HK, Hartemink AJ, \textbf{MacAlpine DM}. Spatiotemporal kinetics of CAF-1-dependent chromatin maturation ensures transcription fidelity during S-phase. Genome Res. 2023 Dec 11. doi: 10.1101/ gr.278273.123. Epub ahead of print. PMC In Process.

\end{enumerate}

\end{statement}

%------------------------------------------------------------------------------
\section{Positions, Scientific Appointments, and Honors}

\subsection*{Positions}
\begin{datetbl}
2022-- & Professor of Biochemistry, Duke University Medical Center\\
2021-- & Professor of Pharmacology and Cancer Biology, Duke University Medical Center \\
2013--      & Director of Graduate Studies in Pharmacology, Duke University Medical Center \\
2013--2020 & Associate Professor of Pharmacology and Cancer Biology, Duke University Medical Center \\
2006--2013  & Assistant Professor of Pharmacology and Cancer Biology, Duke University Medical Center





\end{datetbl}

%2001-2004 	Damon Runyon Cancer Research Foundation Fellowship
%2006		Assistant Professor of Pharmacology and Cancer Biology, Duke Medical %Ctr, Durham, NC
%2006-2011	Whitehead Scholar Award
%2006-2013	Member, Duke Institute for Genome Sciences & Policy
%2007-		Member, Duke Comprehensive Cancer Center
%2009-		Associate Editor, BMC Genomics
%2013-		Executive Committee of Graduate Faculty, Duke University
%2013-		Associate Professor of Pharmacology and Cancer Biology, Duke Medical %Ctr, Durham, NC
%2013		NIH, MGA Ad hoc reviewer
%2014-		Editorial Board, Genome Research
%2014-		American Cancer Society, DMC reviewer
%2015		NIH,  MGA Ad hoc reviewer
%2015-		Director of Graduate Studies in Pharmacology, Duke University
%2016-		NIH, MGA Member

\subsection*{Scientific Appointments}
\begin{datetbl}
2022 & Chair, Duke SOM Bridge Funding Committee\\
2020--2022 & Chair, Duke ITAC (Information, Technology and Computing) \\
2016--2020  & NIH, MGA Member \\
2015    & NIH, MGA Ad hoc Reviewer \\
2014--2016 & American Cancer Society, DMC Reviewer \\
2014-- & Editorial Board, Genome Research \\
2013    & NIH, MGA Ad hoc Reviewer \\
2013--2019    & Executive Committee of Graduate Faculty, Duke University \\
2009--2018           & Associate Editor, BMC Genomics\\
2006--2013     & Member, Duke Institute for Genome Sciences and Policy\\
2007--           & Member, Duke Comprehensive Cancer Center 










\end{datetbl}

\subsection*{Honors}
\begin{datetbl}
2006--2011           & Whitehead Scholar Award \\
2001--2004           & Damon Runyon Cancer Research Foundation Fellowship 



\end{datetbl}

%------------------------------------------------------------------------------

\section{Contributions to Science}

\begin{enumerate}


\item Model Organism Encyclopedia of DNA Elements (modENCODE).

\noindent I was awarded a grant from the National Human Genome Research Institute for the `Systematic Identification and Analysis of Replication Origins in \dros' as part of the model organism ENCODE (modENCODE) initiative. The ENCODE (Encyclopedia of DNA elements) project was a consortium assembled to identify and catalog all functional DNA elements in the human, worm, and fly genomes. I was also a co-PI of the modENCODE data analysis center (DAC) and involved in the systematic and comprehensive analysis of thousands of datasets generated from both the modENCODE and ENCODE consortia.  As a modENCODE data production laboratory, we developed robust protocols and pipelines for the generation and analysis of high-throughput genomic data including ChIP-seq, ChIP-exo, RNA-seq, MNase-seq and DNase-seq.  Together, the consortium generated almost 3,000 publicly available genomic data sets, consisting of array and sequencing based experiments describing the transcription program, mapping the chromatin landscape, identifying transcription factor and insulator binding sites, and characterizing the DNA replication program across multiple cell lines and developmental stages.  The collaborative nature of the consortium environment led to a number of high impact discoveries that span multiple fields and disciplines\textsuperscript{a-d}.

  
\begin{enumerate}
\setlength\itemsep{0.35em}

\item Ho JW, Jung YL, Liu T, Alver BH, Lee S, Ikegami K, Sohn KA, Minoda A, Tolstorukov MY, Appert A, Parker SC, Gu T, Kundaje A, Riddle NC, Bishop E, Egelhofer TA, Hu SS, Alekseyenko AA, Rechtsteiner A, Asker D, Belsky JA, Bowman SK, Chen QB, Chen RA, Day DS, Dong Y, Dose AC, Duan X, Epstein CB, Ercan S, Feingold EA, Ferrari F, Garrigues JM, Gehlenborg N, Good PJ, Haseley P, He D, Herrmann M, Hoffman MM, Jeffers TE, Kharchenko PV, Kolasinska-Zwierz P, Kotwaliwale CV, Kumar N, Langley SA, Larschan EN, Latorre I, Libbrecht MW, Lin X, Park R, Pazin MJ, Pham HN, Plachetka A, Qin B, Schwartz YB, Shoresh N, Stempor P, Vielle A, Wang C, Whittle CM, Xue H, Kingston RE, Kim JH, Bernstein BE, Dernburg AF, Pirrotta V, Kuroda MI, Noble WS, Tullius TD, Kellis M, \textbf{MacAlpine DM}*, Strome S, Elgin SC, Liu XS, Lieb JD, Ahringer J, Karpen GH, Park PJ. Comparative analysis of metazoan chromatin organization. Nature. 2014 Aug 28;512(7515):449-52. PMC4227084. * Co-corresponding author

\item N√®gre N, Brown CD, Ma L, Bristow CA, Miller SW, Wagner U, Kheradpour P, Eaton ML, Loriaux P, Sealfon R, Li Z, Ishii H, Spokony RF, Chen J, Hwang L, Cheng C, Auburn RP, Davis MB, Domanus M, Shah PK, Morrison CA, Zieba J, Suchy S, Senderowicz L, Victorsen A, Bild NA, Grundstad AJ, Hanley D, \textbf{MacAlpine DM}, Mannervik M, Venken K, Bellen H, White R, Gerstein M, Russell S, Grossman RL, Ren B, Posakony JW, Kellis M, White KP. A cis-regulatory map of the Drosophila genome. Nature. 2011 Mar 24;471(7339):527-31. PMC3179250.

\item Kharchenko PV, Alekseyenko AA, Schwartz YB, Minoda A, Riddle NC, Ernst J, Sabo PJ, Larschan E, Gorchakov AA, Gu T, Linder-Basso D, Plachetka A, Shanower G, Tolstorukov MY, Luquette LJ, Xi R, Jung YL, Park RW, Bishop EP, Canfield TK, Sandstrom R, Thurman RE, \textbf{MacAlpine DM}, Stamatoyannopoulos JA, Kellis M, Elgin SC, Kuroda MI, Pirrotta V, Karpen GH, Park PJ. Comprehensive analysis of the chromatin landscape in Drosophila melanogaster. Nature. 2011 Mar 24;471(7339):480-5. PMC3109908.

\item modENCODE Consortium, Roy S, Ernst J, Kharchenko PV, Kheradpour P, Negre N, Eaton ML, Landolin JM, Bristow CA, Ma L, Lin MF, Washietl S, Arshinoff BI, Ay F, Meyer PE, Robine N, Washington NL, Di Stefano L, Berezikov E, Brown CD, Candeias R, Carlson JW, Carr A, Jungreis I, Marbach D, Sealfon R, Tolstorukov MY, Will S, Alekseyenko AA, Artieri C, Booth BW, Brooks AN, Dai Q, Davis CA, Duff MO, Feng X, Gorchakov AA, Gu T, Henikoff JG, Kapranov P, Li R, MacAlpine HK, Malone J, Minoda A, Nordman J, Okamura K, Perry M, Powell SK, Riddle NC, Sakai A, Samsonova A, Sandler JE, Schwartz YB, Sher N, Spokony R, Sturgill D, van Baren M, Wan KH, Yang L, Yu C, Feingold E, Good P, Guyer M, Lowdon R, Ahmad K, Andrews J, Berger B, Brenner SE, Brent MR, Cherbas L, Elgin SC, Gingeras TR, Grossman R, Hoskins RA, Kaufman TC, Kent W, Kuroda MI, Orr-Weaver T, Perrimon N, Pirrotta V, Posakony JW, Ren B, Russell S, Cherbas P, Graveley BR, Lewis S, Micklem G, Oliver B, Park PJ, Celniker SE, Henikoff S, Karpen GH, Lai EC, \textbf{MacAlpine DM}*, Stein LD, White KP, Kellis M. Identification of functional elements and regulatory circuits by Drosophila modENCODE. Science. 2010 Dec 24;330(6012):1787-97. PMC3192495. *Co-corresponding author

\end{enumerate}

\item Chromatin structure and organization in transcription, DNA replication, and repair.   

The local chromatin environment regulates almost all genomic activity including transcription, replication and repair.  We have developed a factor agnostic approach to footprint  protein occupancy on DNA at nucleotide resolution throughout the yeast genome.  This approach provides a holistic and unbiased view of chromatin dynamics at individual loci throughout the genome.  We have used this methodology to characterize origin specific chromatin dynamics during helicase loading\textsuperscript{a} and activation\textsuperscript{b}. We have also developed predictive models of gene expression from the chromatin dynamics associated with exposure to environmental stress\textsuperscript{c}. Finally, we have described the chromatin dynamics that occur following a site specific DNA double strand break and its subsequent repair by non-homologous end joining\textsuperscript{d}.  Together, these studies have provided mechanistic insight into how the local chromatin environment influences a variety of genome activities.  

%the seand activation   a at nucleoMuch progress over the last decade has been made in our understanding of how the local chromatin environment regulates the transcription program.  In contrast, we know comparatively little about how the DNA replication program is regulated by the chromatin landscape.  Our recent work has been focused on understanding how the local chromatin organization impacts the selection and activation of eukaryotic DNA replication origins.  We have found that nucleosome positioning\textsuperscript{a} and nucleosome remodeling\textsuperscript{b,c} at the origin are key determinants regulating the selection and activation of eukaryotic origins.  

\begin{enumerate}
\setlength\itemsep{0.35em}


\item Belsky JA, MacAlpine HK, Lubelsky Y, Hartemink AJ, \textbf{MacAlpine DM}. Genome-wide chromatin footprinting reveals changes in replication origin architecture induced by pre-RC assembly. Genes Dev. 2015 Jan 15;29(2):212-24. PMC4298139.

\item Hoffman RA, MacAlpine HK, \textbf{MacAlpine DM}. Disruption of origin chromatin structure by helicase activation in the absence of DNA replication. Genes Dev. 2021 Oct 1;35(19-20):1339-1355. doi: 10.1101/gad.348517.121. Epub 2021 Sep 23. PMC8494203.

\item Tran TQ, MacAlpine HK, Tripuraneni V, Mitra S, \textbf{MacAlpine DM}, Hartemink AJ. Linking the dynamics of chromatin occupancy and transcription with predictive models. Genome Res. 2021 Jun;31(6):1035-1046. doi: 10.1101/gr.267237.120. Epub 2021 Apr 23. PMC8168580.

\item Tripuraneni V, Memisoglu G, MacAlpine HK, Tran TQ, Zhu W, Hartemink AJ, Haber JE, \textbf{MacAlpine DM}. Local nucleosome dynamics and eviction following a double-strand break are reversible by NHEJ-mediated repair in the absence of DNA replication. Genome Res. 2021 May;31(5):775-788. doi: 10.1101/gr.271155.120. Epub 2021 Apr 2. PMC8092003.

\end{enumerate}


\item Establishment and maintenance of the \dros DNA replication program

In higher eukaryotes, there is little apparent sequence specificity for ORC and the identification of conserved cis-acting elements that direct origin function has remained elusive.  My research program has used \dros to identify the epigenetic determinants of origin selection and function. We have identified activating chromatin marks, nucleosome occupancy, and ATP-dependent chromatin remodelers as being predictive features of ORC binding and origin activation\textsuperscript{a}. We also found that transcription and DNA replication respond to the same chromatin states.  For example, male specific H4K16 hyperacetylation up-regulates transcription and promotes the early replication of the X-chromosome\textsuperscript{b}.  Unlike in mammalian systems, we found that the cell cycle regulated H4K20 monomethylation did not promote helicase loading and origin activation, but rather was essential for maintaining the genomic integrity of late replicating regions of the \dros\ genome\textsuperscript{c}.  Finally, recent work on the loading and distribution of the Mcm2-7 complex revealed that transcription can shape the distribution of the Mcm2-7 complex in late G1, thus providing a possible mechanism(s) for the apparent stochastic activation of replication origins throughout the genomes of higher eukaryotes\textsuperscript{d}.


\begin{enumerate}
\setlength\itemsep{0.35em}

\item Eaton ML, Prinz JA, MacAlpine HK, Tretyakov G, Kharchenko PV, \textbf{MacAlpine DM}. Chromatin signatures of the Drosophila replication program. Genome Res. 2011 Feb;21(2):164-74. PMC3032920.

\item Lubelsky Y, Prinz JA, DeNapoli L, Li Y, Belsky JA, \textbf{MacAlpine DM}. DNA
replication and transcription programs respond to the same chromatin cues. Genome Res. 2014 Jul;24(7):1102-14. PMC4079966.

\item Li Y, Armstrong RL, Duronio RJ, \textbf{MacAlpine DM}. Methylation of histone H4 lysine 20 by PR-Set7 ensures the integrity of late replicating sequence domains in Drosophila. Nucleic Acids Res. 2016 Sep 6;44(15):7204-18.  Epub 2016 Apr 29. PMC5009726.

\item Powell SK, MacAlpine HK, Prinz JA, Li Y, Belsky JA, \textbf{MacAlpine DM}. Dynamic
loading and redistribution of the Mcm2-7 helicase complex through the cell cycle. EMBO J. 2015 Feb 12;34(4):531-43. Epub 2015 Jan 2. PMC4331006.


\end{enumerate}

\item Collaborative work

\noindent A hallmark of my scientific career has been my willingness to collaborate on a diverse array of projects.  We have been delighted to share our expertise in DNA replication, genomics and computational biology with multiple collaborators.  Recent replication focused collaborations include characterizing the role of DNA methylation and heterochromatin in recruiting ORCA/LRWD1 to mammalian replication origins\textsuperscript{a} with Supriya Prasanth and the transcription-dependent displacement of the Mcm2-7 complex in the rDNA of yeast\textsuperscript{b} with Antonio Bedalov.  In collaboration with Robert Duronio, we used genetically engineered histone mutants to perturb constitutive heterochromatin in order to study the impact on the transcription and DNA replication programs\textsuperscript{c}.  Together with Chris Counter, we have been able to apply our molecular biology and computational expertise to develop and extend maximum depth sequencing to identify rare initiating RAS mutations following urethane mutagenesis\textsuperscript{d}.  



\begin{enumerate}
\setlength\itemsep{0.35em}


\item Wang Y, Khan A, Marks AB, Smith OK, Giri S, Lin YC, Creager R, \textbf{MacAlpine DM},
Prasanth KV, Aladjem MI, Prasanth SG. Temporal association of ORCA/LRWD1 to
late-firing origins during G1 dictates heterochromatin replication and
organization. Nucleic Acids Res. 2016 Dec 6. PMC5389698.


\item Foss EJ, Gatbonton-Schwager T, Thiesen AH, Taylor E, Soriano R, Lao U, \textbf{MacAlpine DM}, Bedalov A. Sir2 suppresses transcription-mediated displacement of Mcm2-7 replicative helicases at the ribosomal DNA repeats. PLoS Genet. 2019 May 13;15(5):e1008138. doi: 10.1371/journal.pgen.1008138. PMID: 31083663; PMC6532929.

\item Armstrong RL, Penke TJR, Strahl BD, Matera AG, McKay DJ, \textbf{MacAlpine DM}, Duronio RJ. Chromatin conformation and transcriptional activity are permissive regulators of DNA replication initiation in Drosophila. Genome Res. 2018 Nov;28(11):1688-1700. doi: 10.1101/gr.239913.118. Epub 2018 Oct 2. PMID: 30279224; PMC6211642.


\item Li S, \textbf{MacAlpine DM,} Counter CM. Capturing the primordial Kras mutation initiating urethane carcinogenesis. Nat Commun. 2020 Apr 14;11(1):1800. doi: 10.1038/s41467-020-15660-8. PMC7156420.

\end{enumerate}



\end{enumerate}

\subsection*{Complete List of Published Work in MyBibliography:} 
\medskip

\url{https://www.ncbi.nlm.nih.gov/myncbi/1rU5xQ5uxo75o/bibliography/public/}


%------------------------------------------------------------------------------
\end{document}


\section{Research Support}

\subsection*{Ongoing Research Support}
\medskip

\grantinfo{R35 GM127062-01}
{MacAlpine (PI)}
{04/01/18--03/31/23}
{\it Chromatin-mediated mechanisms of genome integrity}
{The goal of this project is to understand how the local chromatin architecture impacts origin selection, DNA repair and transcription in \scer.}
{Role: PI}

%------------------------------------------------------------------------------

\subsection*{Completed Research Support}
\medskip

\grantinfo{R01 GM104097}
{MacAlpine (PI)}
{08/09/13--08/08/17}
{\it Chromatin architecture defines DNA replication origins}
{The goal of this project is to understand how the local chromatin architecture impacts the selection and activation of eukaryotic DNA replication origins in \scer.}
{Role: PI}

\bigskip

\grantinfo{R01 GM118551}
{Hartemink (PI)}
{04/01/16--03/31/20}
{\it Exploring the role of dynamic chromatin occupancy in transcriptional regulation}
{The goal of this project is to generate predictive models of cell cycle transcription from chromatin occupancy studies.}
{Role: Co-I}

\bigskip

\grantinfo{ACS RSG-11-048-01-DMC}
{MacAlpine (PI)}
{01/01/11--12/31/14}
{\it Defining the Human DNA Replication Program}
{The goal of this project was to establish a comprehensive and genome-wide survey of the human DNA replication program.  We also investigated how the replication program responds to oncogenic transformation.}
{Role: PI}

\bigskip

\grantinfo{NIH U01 HF004279}
{MacAlpine (PI)}
{05/04/07--03/31/12}
{\it The Systematic Identification and Analysis of Replication Origins in Drosophila}
{The major goal of the modENCODE project was to identify all functional DNA elements in a model organism genome.  We  specifically identified and analyzed the sequence elements that direct DNA replication in \dmel.}
{Role: PI}





Chromatin architecture defines DNA replication origins
NIH/NIGMS 1R01GM104097-01
08/09/13-08/08/17
Role: PI
The goal of this project is to understand how the local chromatin architecture impacts the selection and activation of eukaryotic DNA replication origins in S. cerevisiae.  

The role of epigenetics in the formation and consolidation of long-term memories
National University of Singapore
03/01/15-02/29/16
The goal of this project is to profile epigenetic modifications during the the formation and long-term consolidation of memories. 
Role: Co-I

Exploring the Role of Dynamic Chromatin Occupancy in Transcriptional Regulation
NIH/NIGMS R01GM118551 (Hartemink) 04/01/16-03/31/20 1.8 calendar
Role: Co-I
The goal of this project is to generate predictive models of cell cycle transcription from chromatin occupancy data.


Completed

Defining the Human DNA Replication Program
American Cancer Society RSG-11-048-01-DMC (MacAlpine)
01/01/11-12/31/14 (In a no cost extension for 2015)
Role: PI
The goal of this project is to establish a comprehensive and genome-wide survey of the human DNA replication program.  We will also investigate how the replication program responds to oncogenic transformation. 

The Systematic Identification and Analysis of Replication Origins in Drosophila
NIH/NHGRI U01 HG004279
05/04/07-03/31/12
The major goal of the modENCODE project is to identify all functional DNA elements in a model organism genome.  We are specifically identifying and analyzing the sequences elements that direct DNA replication in Drosophila melanogaster.
