\begin{statement}
My research group is focused on understanding how the local chromatin environment impacts fundamental processes like DNA replication, repair and transcription. We utilize a multidisciplinary approach combining genetics, biochemistry, and cell biology with genomics and computational biology to provide fundamental insights into the mechanisms that ensure the faithful and accurate inheritance of genetic and epigenetic information.  We have developed and pioneered genomic approaches to study chromatin occupancy at nucleotide resolution and have used this approach to systematically identify and characterize chromatin dynamics associated with origin selection and activation\textsuperscript{1}, DNA repair\textsuperscript{2}, and the transcriptional response to environmental stress\textsuperscript{3}.  Our work provides a unique look into the 'black box' of chromatin-mediated regulation and rigorously captures single locus chromatin occupancy changes with exquisite resolution.  Coupling our expertise in DNA replication and chromatin occupancy, we have also developed approaches to profile the assembly of chromatin behind the replication fork to better understand the inheritance of epigenetic information\textsuperscript{4}.  

My small research group consists of both experimental and computer scientists and provides rigorous training in the generation and analysis of large genomic data sets. My former trainees have gone on to successful careers in academia and industry.  I have a long standing interest in mentoring students from diverse backgrounds.  To this end, I am the director of graduate studies in Pharmacology, a faculty mentor for Duke's IMSD sponsored BioCORE program, and I have previously mentored an F31 diversity award recipient, Dr. Monica Gutierrez, in my laboratory.

In addition to my group's research activity, I am actively involved in service to the research community.  I frequently participate in peer review and I serve on the editorial board of \textit{Genome Research}.  I have also served as a reviewer for the American Cancer Society (DNA Mechanisms of Cancer) and as a member of the NIH Molecular Genetics A (MGA) study section.  I have also been an \textit{ad hoc} grant reviewer for various special emphasis review panels including multiple phases of NHGRI's ENCODE and the 4D nucleome project. 
\newpage

\begin{enumerate}

\item Hoffman RA, MacAlpine HK, \textbf{MacAlpine DM}. Disruption of origin chromatin structure by helicase activation in the absence of DNA replication. Genes Dev. 2021 Oct 1;35(19-20):1339-1355. doi: 10.1101/gad.34857.121. Epub 2021 Sep 23. PMC8494203.

\item Tripuraneni V, Memisoglu G, MacAlpine HK, Tran TQ, Zhu W, Hartemink AJ, Haber JE, \textbf{MacAlpine DM}. Local nucleosome dynamics and eviction following a double-strand break are reversible by NHEJ-mediated repair in the absence of DNA replication. Genome Res. 2021 May;31(5):775-788. doi: 10.1101/gr.271155.120. Epub 2021 Apr 2. PMC8092003.

\item Tran TQ, MacAlpine HK, Tripuraneni V, Mitra S, \textbf{MacAlpine DM}, Hartemink AJ. Linking the dynamics of chromatin occupancy and transcription with predictive models. Genome Res. 2021 Jun;31(6):1035-1046. doi: 10.1101/gr.267237.120. Epub 2021 Apr 23. PMC8168580. * Co-corresponding author

\item Guti√©rrez MP, MacAlpine HK, \textbf{MacAlpine DM}. Nascent chromatin occupancy profiling reveals locus- and factor-specific chromatin maturation dynamics behind the DNA replication fork. Genome Res. 2019 Jul;29(7): 1123-1133. doi: 10.1101/gr.243386.118. Epub 2019 Jun 19. PMC6633257.

\end{enumerate}

\end{statement}
